wiki/Website-Doc-styling-and-components.md

```diff
@@ -113,7 +113,8 @@ If you want to see these elements in action, have a look at the website itself h
 The `Doc::Badge` component is used to show a textual "badge" with different visual "states".
 
 It provides the following API:
-- `@type` - the type of banner - possible values: `neutral` (default), `information`, `success`, `warning`, `critical`, `outlined`
```

Comment:
Was the 'outlined' type not used anywhere?

Reply:
Nope.